### ISSUE TITLE
Single-line `~H` receives an unwanted newline

### ISSUE THREAD

**Issue by simoncocking** [feliperenan/heex_formatter#48]
e.g.
```
~H"<template hidden></template>"
```
becomes
```
~H"<template hidden></template>
"
```

**Comment by feliperenan** [comment]
Hi @simoncocking 👋🏻 

Thanks for reporting this. I'm working in a refactor to use Inspect.Algebra module to format the code. Hopefully this issue will be solved once I'm done with that.

**Comment by 03juan** [comment]
Unfortunately this is still occurring

**Comment by josevalim** [comment]
This will also need fix in Elixir. I am sending a PR upstream for this soon.

**Comment by josevalim** [comment]
This is fixed but the fix will only work on Elixir v1.13.4 (which has not yet been released).
